hydra-node/exe/tx-cost/Main.hs

```diff
@@ -274,9 +275,13 @@ costOfFanOut = markdownFanOutCost <$> computeFanOutCost
                 <> "| "
                 <> show txSize
                 <> " | "
-                <> show (100 * fromIntegral mem / maxMem)
+                <> show (mem `percentOf` maxMem)
                 <> " | "
-                <> show (100 * fromIntegral cpu / maxCpu)
+                <> show (cpu `percentOf` maxCpu)
                 <> " |"
           )
           stats
+
+percentOf :: (Real a, Real b) => a -> b -> Centi
```

Comment:
nitpick: This does not seem to be related to the PR's purpose, but it's definitely nice :)